Neos.Flow/Tests/Functional/ObjectManagement/Fixtures/PrototypeClassK.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Neos\Flow\Tests\Functional\ObjectManagement\Fixtures;
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * A class to test static compile functionality
+ */
+class PrototypeClassK
+{
+    public function getMicrotime(): float
+    {
+        return static::compiledStaticallyMethod();
+    }
+
+    /**
+     * Method that should get static compiled into the proxy, saving some processing power in production,
+     * but also providing the exact same result on every call.
+     *
+     * @return float
+     * @Flow\CompileStatic
+     */
+    public static function compiledStaticallyMethod(): float
+    {
+        return microtime(true);
```

Comment:
```suggestion
        return bin2hex(random_bytes(10));
```

Reply:
something like this looks fine to you @mhsdesign

Reply:
yeah i have no clue ;) sometimes we also use `md5(uniqid(mt_rand(), true))` but im not a cryptographer ^^ pick what you like :P

Reply:
What about [`Neos\Flow\Utility\Algorithms::generateRandomToken(10)`](https://github.com/neos/flow-development-collection/blob/15591ccdd9bc661765fbe3a25f51de6bb4d7038f/Neos.Flow/Classes/Utility/Algorithms.php#L68C28-L74)?
```suggestion
        return \Neos\Flow\Utility\Algorithms::generateRandomToken(10);
```
(plus a `use` statement)

Reply:
Did you resolve this before of after my comment, @kitsunet?

Reply:
Applied my suggestion while fixing the test…